src/gill/NeckAndShoulderPattern.java

```diff
@@ -0,0 +1,95 @@
+package gill;
+
+import jblockenums.EPattern;
+import jblockmain.*;
+import mathcontainers.Vector2D;
+
+import static java.lang.Math.abs;
+
+public class NeckAndShoulderPattern extends Pattern
+{
+    public NeckAndShoulderPattern(String userName, InputFileData dataStore, MeasurementSet template)
+    {
+        super(userName, dataStore, template);
+    }
+
+    @Override
+    protected EPattern assignPattern()
+    {
+        return EPattern.GILL_NECK_SHOULDER;
+    }
+
+    @Override
+    protected void defineRequiredMeasurements() throws Exception
+    {
+        measurements.addMeasurement(new Measurement("shoulderLengthRight", "A11"));
+        measurements.addMeasurement(new Measurement("frontNeckDiagonal", "A73"));
+        measurements.addMeasurement(new Measurement("backNeckDiagonal", "A74"));
+
+        // TODO: Daniel to finish
+        measurements.addMeasurement(new Measurement("NeckBaseWidth", "A93"));
+        measurements.addMeasurement(new Measurement("ShoulderLengthLeft", "A94"));
+        measurements.addMeasurement(new Measurement("LShoulderPointZ", "A95"));
+        measurements.addMeasurement(new Measurement("RShoulderPointZ", "A96"));
+        measurements.addMeasurement(new Measurement("LSideNeckPointZ", "A97"));
+        measurements.addMeasurement(new Measurement("RSideNeckPointZ", "A98"));
+    }
+
+    @Override
+    public void createBlocks()
+    {
+        // Pull from store
+        var shoulderLengthRight = get("shoulderLengthRight");
+        var frontNeckDiagonal = get("frontNeckDiagonal");
+        var backNeckDiagonal = get("backNeckDiagonal");
+
+        // TODO: Daniel to finish parsing measurements into local variables
+        var NeckBaseWidth = get("NeckBaseWidth");
+        var ShoulderLengthLeft = get("ShoulderLengthLeft");
+        var LShoulderPointZ = get("LShoulderPointZ");
+        var RShoulderPointZ = get("RShoulderPointZ");
+        var LSideNeckPointZ = get("LSideNeckPointZ");
+        var RSideNeckPointZ = get("RSideNeckPointZ");
+
+        // Build block
+        Block fullBlock = new Block(userName + "_Gill_NeckShoulder");
+        blocks.add(fullBlock);
+
+        //If conditions
+        double Rsidepoint = ((RSideNeckPointZ < 0 &&  RShoulderPointZ < 0) || (RSideNeckPointZ > 0 &&  RShoulderPointZ < 0)) ? abs(RSideNeckPointZ - RShoulderPointZ) : -abs(RSideNeckPointZ - RShoulderPointZ);
```

Comment:
Can you make your variable naming consistent? The convention elsewhere is the software is to start with a lower case and use camel-case for the reset so these ought to be "sidePointRight". Same for lines 49-52. See line 48 as an example.

Reply:
variables made more consistent